src/test/kotlin/org/rust/lang/core/RsControlFlowGraphTest.kt

```diff
@@ -0,0 +1,260 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.lang.core
+
+import junit.framework.ComparisonFailure
+import org.intellij.lang.annotations.Language
+import org.rust.lang.RsTestBase
+import org.rust.lang.core.psi.RsBlock
+import org.rust.lang.core.psi.ext.descendantsOfType
+
+class RsControlFlowGraphTest : RsTestBase() {
```

Comment:
There are very few tests. Especially, there are no tests for loop expression and loop expression with breaks. I.e. loop expr without breaks is really an infinite loop and should be the last node in CFG.

Try to run these tests with coverage and ensure all methods in CFGBuilder are executed